Library/models.py

```diff
@@ -1,6 +1,6 @@
 from flask_login import UserMixin
 
-from Library_Management_System import db
+from Library import db

```

Comment:
Any reason to rename the directory structure, the earlier one was more clear

Reply:
Sorry for that, you can keep the original 

It was just the project folder have same name as a folder inside(that have most of code) it so I felt it a bit confusing every time I have to keep track which folder I am in actually